znai-reactjs/src/doc-elements/structure/links.js

```diff
@@ -22,7 +22,16 @@ function isLocalUrl(url) {
         return false
     }
 
-    return url.startsWith('/' + getDocId())
+    return url.startsWith('#') ||
+        url.startsWith('/' + getDocId())
+}
+
+function isExternalUrl(url) {
```

Comment:
Can yo rename it as isInternalUrl, then you don't have to use a negate operator?

Reply:
this one is tricky. We already have `isLocalUrl` but that refers to a path having `/<doc-id>` in front. So the best I could come up to avoid negating was `isNotLocalUrl` :)

Reply:
Then forget it.